controllers.js

```diff
@@ -17,8 +17,28 @@ exports.fetchTopics = (req, res, next) => {
     })
     .catch(next)
 };
+exports.fetchArticle = (req, res, next) => {
+  const { article_id } = req.params;
 
+  
+  if (isNaN(article_id)) {
+      return res.status(400).send({ message: 'Bad Request!' });
+  }
+
+  getArticleById(article_id)
+      .then(article => {
+          if (!article) {
+              return res.status(404).send({ message: 'Article not found' });
```

Comment:
see above re. error handling middleware.  we want to create a custom error here with promise.reject()